packages/react-components/components/BackButton.v2.tsx

```diff
@@ -0,0 +1,35 @@
+import BackChevron from '@celo/react-components/icons/BackChevron.v2'
+import colors from '@celo/react-components/styles/colors.v2'
+import variables from '@celo/react-components/styles/variables'
+import * as React from 'react'
+import { StyleSheet, TouchableOpacity, TouchableOpacityProps, View } from 'react-native'
+
+type Props = TouchableOpacityProps & {
+  height?: number
+  color?: string
+  testID?: string
+}
+export default function BackButton({ onPress, color, height, disabled, testID }: Props) {
+  return (
+    <View style={styles.container}>
+      <TouchableOpacity
+        hitSlop={variables.iconHitslop}
+        onPress={onPress}
+        disabled={disabled}
+        testID={testID}
+      >
+        <BackChevron height={height ? height : SIZE} color={color ? color : colors.greenBrand} />
+      </TouchableOpacity>
+    </View>
+  )
+}
+
+const SIZE = 16
+
+const styles = StyleSheet.create({
+  container: {
+    paddingLeft: variables.contentPadding + 7,
```

Comment:
im not sure about using these variables in new design. since everything is multiples of 8. the +7 worries me

Reply:
+1, setting up new variables may be nice.
But also, I find variables for things like padding to be unhelpful. How often will we say 'reduce padding across the app by 2'. Even if we did, unlikely that most things would line up right after the var change

Reply:
I've moved this logic to `mobile` package. Regarding the padding itself I've changed it to `+6` (correlates with the design) to the standard padding we have.